packages/radix-ui-themes/src/styles/tokens/color.css

```diff
@@ -282,332 +126,153 @@
     --color-panel: var(--color-panel-translucent);
+    --backdrop-filter-panel: blur(64px);
   }
 }
 
 /* * * * * * * * * * * * * * * * * * * */
 /*                                     */
-/*           Step 9 contrast           */
-/*                                     */
-/* * * * * * * * * * * * * * * * * * * */
-
-:root {
-  --tomato-9-contrast: white;
-  --red-9-contrast: white;
-  --ruby-9-contrast: white;
-  --crimson-9-contrast: white;
-  --pink-9-contrast: white;
-  --plum-9-contrast: white;
-  --purple-9-contrast: white;
-  --violet-9-contrast: white;
-  --iris-9-contrast: white;
-  --indigo-9-contrast: white;
-  --blue-9-contrast: white;
-  --cyan-9-contrast: white;
-  --teal-9-contrast: white;
-  --jade-9-contrast: white;
-  --green-9-contrast: white;
-  --grass-9-contrast: white;
-  --orange-9-contrast: white;
-  --brown-9-contrast: white;
-  --sky-9-contrast: #1c2024; /* light mode slate 12 */
-  --mint-9-contrast: #1a211e; /* light mode sage 12 */
-  --lime-9-contrast: #1d211c; /* light mode olive 12 */
-  --yellow-9-contrast: #21201c; /* light mode sand 12 */
-  --amber-9-contrast: #21201c; /* light mode sand 12 */
-  --gold-9-contrast: white;
-  --bronze-9-contrast: white;
-  --gray-9-contrast: white;
-}
-
-/* * * * * * * * * * * * * * * * * * * */
-/*                                     */
-/*          Gray 2 equivalent          */
-/*        for translucent panels       */
+/*            Accent color             */
 /*                                     */
 /* * * * * * * * * * * * * * * * * * * */
 
-.dark,
-.dark-theme {
-  --gray-2-translucent: #1d1d1db3;
-  --mauve-2-translucent: #1e1d1eb3;
-  --slate-2-translucent: #1b1d1eb3;
-  --sage-2-translucent: #1a1c1bb3;
-  --olive-2-translucent: #1b1c1ab3;
-  --sand-2-translucent: #1d1d1bb3;
-
-  @supports (color: color(display-p3 1 1 1)) {
-    @media (color-gamut: p3) {
-      & {
-        --gray-2-translucent: color(display-p3 0.1137 0.1137 0.1137 / 0.7);
-        --mauve-2-translucent: color(display-p3 0.1176 0.1137 0.1176 / 0.7);
-        --slate-2-translucent: color(display-p3 0.1059 0.1137 0.1176 / 0.7);
-        --sage-2-translucent: color(display-p3 0.102 0.1137 0.1059 / 0.7);
-        --olive-2-translucent: color(display-p3 0.1059 0.1137 0.102 / 0.7);
-        --sand-2-translucent: color(display-p3 0.1137 0.1137 0.1059 / 0.7);
-      }
-    }
-  }
-}
-
-/* * * * * * * * * * * * * * * * * * * */
-/*                                     */
-/*           Surface colors            */
-/*                                     */
-/* * * * * * * * * * * * * * * * * * * */
+[data-accent-color='amber'] {
+  --accent-1: var(--amber-1);
+  --accent-2: var(--amber-2);
+  --accent-3: var(--amber-3);
+  --accent-4: var(--amber-4);
+  --accent-5: var(--amber-5);
+  --accent-6: var(--amber-6);
+  --accent-7: var(--amber-7);
+  --accent-8: var(--amber-8);
+  --accent-9: var(--amber-9);
+  --accent-10: var(--amber-10);
+  --accent-11: var(--amber-11);
+  --accent-12: var(--amber-12);
```

Comment:
The steps still have to remapped here in `color.css`, otherwise components that use a color that’s not imported will fall back to the latest defined `--accent` variables, which felt weird.

Reply:
and tbh I believe there was another issue with that but I forgot what it was 😅

Reply:
The remaps are about 30 KB unminified ungzipped

Reply:
hmm, that's a bit annoying, I wonder if we could do it differently with a fallback instead? So if not defined, it would fallback to some annoying color (would also help realising you need to import a missing color too).

Reply:
OK I remembered the real reason actually, it's needed for realiasing colors